Assets/Scripts/Framework/GeoLocation/RouteHighlighter.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using System.Collections.Generic;
+using System.Collections;
+using UnityEngine;
+
+namespace Framework.GeoLocation
+{
+    public sealed class RouteHighlighter : MonoBehaviour
```

Comment:
```suggestion
{
    [RequireComponent(typeof(LineRenderer))]
    public sealed class RouteHighlighter : MonoBehaviour
```

Je zou ook een check doen in `Awke` of deze refrence gevulled is, zo niet `GetComponent`.